Dockerfile

```diff
@@ -144,18 +174,7 @@ COPY package.json /usr/src/app/
 # Install only the production modules that have C extensions
 # Save the modules and then install devDependencies for build
```

Comment:
I think this line needs to be removed now as it seems devDeps are installed in `noed-build`.